### ISSUE TITLE
[BUG] The markers for Titans on the minimap don't update when the Titan is killed

### ISSUE THREAD

**Issue by Acros5542** [AoTTG-2/AoTTG-2#503]
**Describe the bug**
This is a minor bug, as it exists in AoTTG1 as well, but on the minimap, the indicator for a Titan's location (the yellow circle) does not disappear immediately upon the titan dying, instead disappearing only once the Titan's model has disappeared. Only reason I'm reporting it is since the counter for remaining titans now updates immediately upon the titan dying rather than despawning, it may be worth making the minimap match for the sake of consistency.

**To Reproduce**
1. Go to any map/gamemode with Titans
2. Spawn in as a human and kill one
3. Look at the minimap and notice when the yellow dot indicating the Titan's position disappears

**Expected behavior**
If wanting to match the kill feed at the top of the screen, the yellow dot indicator should disappear at the same time as the Titan is killed rather than unloaded from the level

**Contributor instructions**
1. Try to recreate the bug via the reported reproduction steps. If the bug can be reproduced, remove the investigation label. If the bug cannot be reproduced, please add a comment, attach "On Hold" label and inform the submitter that the bug cannot be reproduced. If there is no reply after 2 weeks, the issue can be closed.
2. Follow the default development team procedures (Create a new branch, apply your fix, create a pull request, submit new builds for testing team in dev squad)
3. Once testing team has confirmed the issue is resolved and the pull request is accepted, the issue can be closed.

**Comment by DaSquyd** [comment]
Might as well add this one too lol
